### PR TITLE
obs-studio-plugins: normalize

### PR DESCRIPTION
**Pull request by superherointj**
obs-studio-plugins: normalize

* I plan doing individual refactor of plugins in separate. (On updates, maybe.)

### CODE REVIEW

File: pkgs/applications/video/obs-studio/plugins/default.nix [hunk 1]
```diff
@@ -1,18 +1,34 @@
 { callPackage, libsForQt5, pkgsi686Linux }:
 
+# When adding new plugins:
+# - Respect alphabetical order. On diversion, file a PR.
+# - Feel free to *not* prefix packages with 'obs-'.
```
[the remark this answers is not in the document]
Comment (superherointj) [reply pkgs/applications/video/obs-studio/plugins/default.nix:5]: The wording does not imply this. Was trying to say that `obs-` is not mandatory.

[the remark this answers is not in the document]
Comment (superherointj) [reply pkgs/applications/video/obs-studio/plugins/default.nix:5]: > - Package name should reflect upstream package name. Including or excluding `obs` prefix/suffix.

?

[the remark this answers is not in the document]
Comment (superherointj) [reply pkgs/applications/video/obs-studio/plugins/default.nix]: @06kellyjac I have updated the wording.

[the remark this answers is not in the document]
Comment (06kellyjac) [reply pkgs/applications/video/obs-studio/plugins/default.nix]: LGTM

**Review by SuperSandro2000 — APPROVED** [review]

**Pull request merged**